dali/python/nvidia/dali/reducers.py

```diff
@@ -90,7 +93,9 @@ def cell_unpickle():
 
 
 def cell_reducer(cell):
-    return (cell_unpickle, tuple(), {'cell_contents': cell.cell_contents}, None, None, set_cell_state)
+    return (cell_unpickle, tuple(), {
+        'cell_contents': cell.cell_contents
+    }, None, None, set_cell_state)
```

Comment:
I think this reads a bit better...
```suggestion
    return (cell_unpickle, tuple(),
        {'cell_contents': cell.cell_contents},
        None, None, set_cell_state)
```